packages/webdriverio/src/utils/interception/devtools.ts

```diff
@@ -100,7 +107,10 @@ export default class DevtoolsInterception extends Interception {
                 /**
                  * no stubbing if no overwrites were defined
                  */
-                if (mock.respondOverwrites.length === 0) {
+                if (mockResponded || mock.respondOverwrites.length === 0) {
+                    mock.emit('match', request)
```

Comment:
Not emitting this event on L105 so we would have `request` with `mockedResponse` on overwrite